build.gradle

```diff
@@ -3,21 +3,24 @@ plugins {
     id 'maven-publish'
     id 'signing'
     id 'jacoco'
+    id 'io.freefair.lombok' version '6.5.0.3'
 }
 
 repositories {
-    mavenLocal()
-    maven {
-        url = uri('https://repo.maven.apache.org/maven2/')
-    }
+    mavenCentral()
 }
 
 dependencies {
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.2.2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.3'
     implementation 'org.slf4j:slf4j-api:1.7.36'
-    testImplementation 'org.junit.platform:junit-platform-runner:1.8.2'
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
 
+    testImplementation 'org.junit.platform:junit-platform-runner:1.9.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.9.0'
+    testImplementation 'ch.qos.logback:logback-classic:1.2.11'
```

Comment:
I have added this to have proper logging when the tests are running as:
- it can help with debugging of test issues as logs are produced for tests
- it eliminates the warning produced by SLF4J when it is not able to find related configuration like this:
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

Reply:
Hm, OK. At least, the second issue could fix `org.slf4j:slf4j-nop:...` dependency.
But if it is useful for debugging then let's leave it for now.